plugins/org.eclipse.sirius.tests.junit/src/org/eclipse/sirius/tests/unit/common/migration/DiagramMigrationTestCampaign02.java

```diff
@@ -114,12 +114,12 @@ public void testEdgeLayout() throws Exception {
         openEditorOnDiagram("" + diagramID);
```

Comment:
Copyright of the file must be shift to 2023

Reply:
done